dll/cpl/mmsys/sounds.c

```diff
@@ -935,7 +935,7 @@ ShowSoundScheme(PGLOBAL_DATA pGlobalData, HWND hwndDlg)
             tvItem.item.state = TVIS_EXPANDED;
             tvItem.item.stateMask = TVIS_EXPANDED;
             tvItem.item.pszText = pLabelMap->szDesc;
-            if (pLabelContext->szValue && _tcslen(pLabelContext->szValue) > 0)
+            if (pLabelContext->szValue[0] && _tcslen(pLabelContext->szValue) > 0)
```

Comment:
```suggestion
            if (pLabelContext->szValue[0])
```
Both do more or less the same (tcslen checks more than 1 character, but that should not matter here.)